README.md

```diff
@@ -27,6 +29,67 @@ Scripts to create and load data into the `gxa-*` Solr indexes (for analytics). E
 - jq (1.5)
 - curl
 
+However, the direct usage without `atlas-index-base:1.5` container (or the container tag currently used in the CI tests on [run_tests_in_containers.sh](run_tests_in_containers.sh)) is discouraged.
+
+# Authentication
+
+The setup on the CI is made to use authentication with default user and password. The calls assume these settings (solr:SolrRocks), but the user and password can be modified by doing:
+
+```
+export SOLR_USER=<new-user>
+export SOLR_PASS=<new-pass>
+```
+
+To use default auth in a new solr cloud instance, upload `test/security.json` to ZK as shown in the `Setup auth` part of the `run_tests_in_containers.sh`. To setup users in a production setting the script [create-users.sh](bin/create-users.sh) will receive two set of users:
```

Comment:
Typo: “SolrCloud”. I wouldn’t point it out but I’ve seen you’ve gone the extra mile to write it correctly elsewhere. :)